kie-server-parent/kie-server-services/kie-server-services-prometheus/src/main/java/org/kie/server/services/prometheus/PrometheusJobListener.java

```diff
@@ -87,6 +101,13 @@ public void afterJobExecuted(AsynchronousJobEvent event) {
         final RequestInfo job = event.getJob();
         numberOfJobsExecuted.labels(defaultString(job.getDeploymentId()), String.valueOf(event.failed()), job.getCommandName()).inc();
         numberOfRunningJobs.labels(defaultString(job.getDeploymentId()), job.getCommandName()).dec();
+        if(job.getRetries() > 0 && job.getStatus().equals(STATUS.RETRYING)) {
+        	numberOfJobsRetrying.labels(defaultString(job.getDeploymentId()), String.valueOf(event.failed()), job.getCommandName()).inc();
+        }
+        else if(job.getRetries() == 0 && job.getStatus().equals(STATUS.ERROR)) {        	
+        	numberOfJobsErrored.labels(defaultString(job.getDeploymentId()), String.valueOf(event.failed()), job.getCommandName()).inc(); 
+        	numberOfJobsRetrying.labels(defaultString(job.getDeploymentId()), String.valueOf(event.failed()), job.getCommandName()).dec();
```

Comment:
This is not correct from my point of view (if this metric means the :
- with 0 retries, it will obtain a negative number
- with N retries (10 for example), after the command failed the maximum number of retries, the status is ERROR and should be decreased once (it's related to the previous comment: for the same job, it has to be 1 at maximum)